buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -131,6 +131,7 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
          'aarch64_linux_xl_cm': CURRENT_RELEASES,
          'aarch64_linux_xl_uma': CURRENT_RELEASES,
          'aarch64_linux_mixed' : CURRENT_RELEASES,
+         'aarch64_mac' : ['17'],
```

Comment:
Please add `'18'` here.

Reply:
I opened PRs for JDK 11 and JDK next.  No support for JDK 8 yet.
- https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/474
- https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/398

So this line can be like `'aarch64_mac' : CURRENT_RELEASES - '8'` ?

Reply:
It can just be `CURRENT_RELEASES` since we intend to support 8 later.